### PR TITLE
Created Home page.

### PR DESCRIPTION
**Pull request by j24m**
Created home page for e-com website.

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]
Code looks fine. Good UI 👍

File: style.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    gap: 1rem;
+    padding: 1rem;
+    font-size: 4rem;
+    font-family: "Dancing Script";
+    font-weight: 700;
+}
+
+.homepage-nav-logo:hover {
+    transform: scale(1.001);
+    text-shadow: 2px 2px 5px black;
```
Comment (zoef-yt) [inline style.css]: use var for colours

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks absolutely amazing. Just add a preview to your UI in the description of the pr with video, gif, or photos. Nice use of keyframes and animation. Good to go. Good to merge.

**Comment by bhatvikrant** [comment]
Looks good @j24m 
but I think the text on homepage is not in center

Screenshot 2022-02-13 at 2 03 11 PM

**Review by bhatvikrant — APPROVED** [review]

**Review by saurabhchirde — APPROVED** [review]
Loved the animation and UI. Code is clean and readable. GTM.
Just a suggestion, on the landing page the text and button are not center-aligned.

Screenshot 2022-02-13 at 2 12 15 PM

**Pull request merged**